### DISCUSSION TITLE
Gracenote 2

### DISCUSSION THREAD

**Discussion by shap4ever** [harman-f/MHI2_MIB2_AIO_FW_Update_Template#25] [General]
Hi,
First of all many thanks for your great job, i have a question about the Gracenote2 part during prepration of the installation media, Shall I use the updated "Gracenote2 EU-v24-0010-1.1.1.3119-20231019~20240215-191926.7z" instead of "Gracenote2_AIO_payload.7z" which is dated 2023? I see the file sizes are not similar too. If the answer is no how I can update it afterward? I also want to ask about the possibility of doing this AIO update while the engin is running (without start-stop)
I wish you a great day and thanks in advance for your help.

**Comment by Fanaticjonathanjordan** [comment]
Hello, good afternoon or evening always supporting your work but in my case I copied the files as the instructions said I have the vw firmware (MHI2_ER_VWG11_K3342) and I get error N/A|mainUnit: EM2-H-N-EU-VW-MQB copy Gracenote2_AIO_payload.7z do I have to copy the Gracenote2 EU-V24 version? Greetings…
